src/test/test_verify.cpp

```
@@ -541,6 +541,7 @@ TEST_PROGRAM("build", "prog_array.o", ".text", "func0", 5);
 TEST_PROGRAM("build", "prog_array.o", ".text", "func1", 5);
 TEST_PROGRAM("build", "prog_array.o", ".text", "func2", 5);
 TEST_PROGRAM("build", "prog_array.o", ".text", "func3", 5);
+TEST_SECTION("build", "externalfunction.o", ".text");
```

Clang (14.0.6)

[warning] variable 'TEST_CASE' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)